src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/SecondHop.tsx

```diff
@@ -104,19 +107,21 @@ export const SecondHop = ({
       bn(buyAmountCryptoPrecision).times(buyAssetFiatRate).toString(),
     )
 
+    const tradeTx =
```

Comment:
`tradeTxHash` to make it `txHash,` instead of `txHash: txId` below?
Also, this seems to be a Txid vs. a Tx hash (missing the 0x), is this intended or only a case of the mock being like so for now?

Reply:
Second hop is out of datae and is fixed in later PRs